### PR TITLE
Added timezone under location. Fixes #10

### LINKED ISSUE #10: Change state.lastchange from unix timestamp to rfc3339 (JSON schema date-time)
I would like to change `state.lastchange` type from number to date-time.

The problem lies within that a simple number does not carry timezone information with it. As there is no timezone field specified in the API, apps cannot determine in which timezone a space lives.

JSON Schema defines the date-time type, which is the rfc3339 format.

I think also that in future all time related properties should be date-time type, if there are added

### PR DESCRIPTION
**Pull request by the-metalgamer**

### CODE REVIEW

File: 14-draft.json [hunk 1]
```diff
@@ -36,6 +36,10 @@
         "lon": {
           "description": "Longitude of your space location, in degree with decimal places. Use positive values for locations west of Greenwich, and negative values for locations east of Greenwich.",
           "type": "number"
+        },
+        "timezone": {
+          "description": "The timezone the space is located in. It should be formatted according to the <a target=\"_blank\" href=\"https://en.wikipedia.org/wiki/List_of_tz_database_time_zones\">TZ database location names</a>.",
```
Comment (rnestler) [inline 14-draft.json]: Is there a more "formal" specification than Wikipedia?

Comment (the-metalgamer) [inline 14-draft.json]: There is https://www.iana.org/time-zones , but they don't provide an HTML site, where all the names are listed.

**Comment by the-metalgamer** [comment]
Rebased against master.

**Comment by dbrgn** [comment]
> Or should we do it with the merge?

I'd always try to do squash-merges if possible.

I think Wikipedia is fine. Ok if we merge, @rnestler?

**Pull request merged**